### DISCUSSION TITLE
Change hostname for Sensor Node

### DISCUSSION THREAD

**Discussion by Jaap79** [Security-Onion-Solutions/securityonion#11517] [Unsupported Versions] [answered]
Hi,

We've recently deployed a sensor node but it turns out there was a typo in the naming convention. This isn't earth shattering by any means but with most nodes having naming conventions like 'xxx-yyyy-zzzz' and one node having 'xxy-yyy-zzzz' it makes building queries more of a nuisance, and it triggers some OCD when looking at the environment as a whole ;)

Is there any way to change the host name of a sensor node after deploying the SO software? It's already generating valuable data and it's in a bit of a remote site which makes it more difficult to reinstall at this time.

**Comment by defensivedepth** [comment, marked as answer]
According to our documentation (https://docs.securityonion.net/en/2.4/installation.html):

> Warning
> Please make sure that your hostname is correct during installation. Setup generates certificates based on the hostname and we do not support changing the hostname after Setup.
